### ISSUE TITLE
Order Creation

### ISSUE THREAD

**Issue by amenk** [LimeSoda/LimeSoda_SampleDataGenerator#11]
What about creating orders?

This would be a good start:

https://gist.github.com/ajzele/2781404

**Comment by mzeis** [comment]
Would be possible. I'm in the mid of a refactoring right now; it may be better to think about adding new features when v0.2 is released.

**Comment by philwinkle** [comment]
An additional request may be to add sales order data generator for testing of EE features like customer segments,  benchmarking purposes.

**Comment by philwinkle** [comment]
Yes of course - to be sure I have created generators that do this to some
degree; but to create orders on the order of 100's of thousands would
likely take _quite_ a while. Perhaps I was thinking a more direct,
insert-based approach (though that may be outside the bounds of the
limesoda generator)

On Mon, Sep 9, 2013 at 9:50 AM, Alexander Menk notifications@github.comwrote:

> What about creating orders?
> 
> This would be a good start:
> 
> https://gist.github.com/ajzele/2781404
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/LimeSoda/LimeSoda_SampleDataGenerator/issues/11
> .

## 

Follow me on Twitter - @philwinkle
Friend me on Facebook - http://www.facebook.com/philwinkle

**Comment by mzeis** [comment]
Exactly. We don't want to use raw SQL statements. The goal of v0.2 will be to be more flexible in creating the test data. While I can't promise it may be possible that you write your own adapter to insert data rows directly without using Magento models.

**Comment by philwinkle** [comment]
While I'm anti-raw-sql as well, a generator where one could create a more performant insert would be fantastic -- I look forward to v0.2 to see if this would be possible. 

Cheers.
